etna/core/models/basepage.py

```diff
@@ -49,12 +49,37 @@ class BasePage(MetadataPageMixin, DataLayerMixin, Page):
         blank=True,
         on_delete=models.SET_NULL,
         related_name="+",
+        help_text=_("Image that will appear on thumbnails and promos around the site."),
     )
 
     # DataLayerMixin overrides
     gtm_content_group = "Page"
 
-    promote_panels = MetadataPageMixin.promote_panels + [
+    # Overriding the original help_text set by default in MetadataPageMixin
```

Comment:
Suggestion: Add a note here explaining that you're overidding the wagtail core model Page's promote_panels, this is fine now but in the future if there's a change in Wagtail it won't be automatically reflected here due to us redefining these field panels.

Reply:
There's an option here to pop the fields out of the promote panels, (it's a list after all). It's awkward but something to consider as that negates the above point.

Reply:
I did think of this earlier... I was thinking I could do a for loop through the `MetadataPageMixin.promote_panels` and then when it hits `slug`, `search_description`, or `search_image` - edit the `help_text` accordingly. This would negate the issue of the MetadataPageMixin being updated and then those changes not being displayed here.

Is there a better way to do this?

Reply:
@jamesbiggs I wouldn't worry about this too much. It's a pain to do things like this with multi-level panels. Anything you do end up with will not be as clear as what you have here.

In a previous project, I invented a new `PanelSet` entity that was used instead of the lists, and had the powers to access panels by name regardless of structure, amongst other things. But, there's no need for that here.